go/app/infra.go

```diff
@@ -20,6 +29,8 @@ type Item struct {
 //go:generate go run go.uber.org/mock/mockgen -source=$GOFILE -package=${GOPACKAGE} -destination=./mock_$GOFILE
 type ItemRepository interface {
 	Insert(ctx context.Context, item *Item) error
+	GetAllItem(ctx context.Context) ([]Item, error)
```

Comment:
[nits]
複数データを返却する場合は、命名でそれを示してあげると良いです :+1:

```suggestion
	GetAllItems(ctx context.Context) ([]Item, error)
```

API命名に関してはこの本([ref](https://www.amazon.co.jp/dp/4873116864/))あたりが参考になります